platform/broadcom/sonic-platform-modules-juniper/qfx5210/utils/juniper_qfx5210_util.py

```diff
@@ -531,26 +530,24 @@ def device_traversal():
     if len(ALL_DEVICE)==0:
         devices_info()
     for i in sorted(ALL_DEVICE.keys()):
-        print("============================================")        
+        print("============================================")
         print(i.upper()+": ")
         print("============================================")
          
         for j in sorted(ALL_DEVICE[i].keys(), key=get_value):    
-            print "   "+j+":",
```

Comment:
>, [](http://example.com/codeflow?start=29&length=1)

Original print without newline, the new code changes behavior.

Reply:
Strictly speaking, you are using a blank as end, you should use empty string.

Reply:
I think blank space should be used, ref: https://stackoverflow.com/a/18908914/19880750

Reply:
Agree.